pinakes/main/models.py

```diff
@@ -62,6 +62,7 @@ class State(models.TextChoices):
         """states for Source"""
 
         DONE = "Done"
+        INPROGRESS = "InProgress"
```

Comment:
@hsong-rh Should this name be IN_PROGRESS?

Reply:
Also since this is going to be displayed to the end user should they have localization enabled.